gulpfile.js

```diff
@@ -61,15 +65,21 @@ function manual() {
 }
 
 function reference() {
-  return gulp
-    .src('src/reference/**/*.md', {base: 'src'})
+  return merge(
+    gulp.src(['build/reference/*/references/*.json'], {base: 'build'})
+      .pipe(raw()),
+    merge(
```

Comment:
It's just this way because we do different things to the first `gulp.src` vs the second two. The first one only gets passed to `raw()` but the second two have three operations done to them. Finally all files have the final 4 operations done.